src/views/ProtestPage.js

```diff
@@ -77,7 +77,15 @@ function useFetchProtest() {
 
 function getFutureDates(dateTimeList) {
   if (dateTimeList?.length) {
-    return dateTimeList.filter((dateTime) => new Date(dateTime.date) >= new Date());
+    return dateTimeList.filter((dateTime) => {
+      const now = new Date();
+      const today = new Date(Date.UTC(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate()));
+
+      const demoDate = new Date(dateTime.date);
```

Comment:
Would `protestDate` be clearer for this?

Reply:
yes I will fix that